libraries/AC_AttitudeControl/AC_AttitudeControl_Multi_6DoF.cpp

```diff
@@ -149,15 +149,15 @@ void AC_AttitudeControl_Multi_6DoF::input_angle_step_bf_roll_pitch_yaw(float rol
 // Command a Quaternion attitude with feedforward and smoothing
 // attitude_desired_quat: is updated on each time_step (_dt) by the integral of the angular velocity
 // not used anywhere in current code, panic in SITL so this implementation is not overlooked
-void AC_AttitudeControl_Multi_6DoF::input_quaternion(Quaternion& attitude_desired_quat, Vector3f ang_vel_target) {
+void AC_AttitudeControl_Multi_6DoF::input_quaternion(Quaternion& attitude_desired_quat, Vector3f ang_vel_body) {
```

Comment:
should be:
 const Vector3f &ang_vel_body

Reply:
Done

Reply:
Changed it and realised that the input quaternion modifies the parameter on the stack. Happy to fix this on all the code I am responsible for in a single PR.